packages/react-scripts/config/paths.js

```diff
@@ -54,7 +54,7 @@ module.exports = {
   appBuild: resolveApp('build'),
   appPublic: resolveApp('public'),
   appHtml: resolveApp('public/index.html'),
-  appIndexJs: resolveApp('src/index.js'),
+  appIndexJs: resolveApp('src/index'),
   appPackageJson: resolveApp('package.json'),
   appSrc: resolveApp('src'),
   testsSetup: resolveApp('src/setupTests.js'),
```

Comment:
@detrohutt: consider removing the `.js` extension on `src/setupTests` to generify this file as well (3 instances total in this `packages/react-scripts/config/paths.js`).